File: util/photo_google.py
from apiclient.discovery import build
import os
def getPhotos(nome):
    service = build("customsearch", "v1",
                os.environ.get("developerKey"))
    res = service.cse().list(
        q=nome,
        cx=os.environ.get("cx"),
        searchType='image',
        num=3,
        imgType='photo',
        fileType='png',
        safe= 'off'
    ).execute()

    if not 'items' in res:
        return []
    else:
        return [{'title':item['title'], 'link':item['link']} for item in res['items']]
            
if __name__ == "__main__":
    for item in getPhotos("maradona"):
        print(f"{item['title']}  {item['link']}")

File: server/services/photo_google.js
'use strict';

const {google} = require('googleapis');
const customsearch = google.customsearch('v1');

const  getPhotos=(nome)=>{
    const res = await customsearch.cse().list({
        q:nome,
        cx:process.env.get("cx"),
        searchType:'image',
        num:3,
        imgType:'photo',
        fileType:'png',
        safe:'off',
        auth:process.env.get("developerKey")
        }).execute();

        return res.data;

}        

module.exports = getPhotos;